src/LogExpert.UI/Controls/LogWindow/TimeSpreadigControl.cs

```diff
@@ -145,14 +145,14 @@ private void OnLineSelected (SelectLineEventArgs e)
 
     #region Events handler
 
-    private void TimeSpreadCalc_CalcDone (object sender, EventArgs e)
+    private void OnTimeSpreadCalcCalcDone (object sender, EventArgs e)
     {
         _logger.Debug(CultureInfo.InvariantCulture, "timeSpreadCalc_CalcDone()");
 
         lock (_monitor)
         {
             Invalidate();
-            Rectangle rect = ClientRectangle;
+            var rect = ClientRectangle;
             rect.Size = new Size(rect.Width, rect.Height - _edgeOffset * 3);
 
             if (rect.Height < 1)
```

Comment:
gfx and brush should be with using.
Also, this class should implement IDisposable for things like _bitmap.